bundle/manifests/projectcontour-contour-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,350 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: projectcontour-contour-operator.v0.0.0
+  namespace: placeholder
```

Comment:
Not sure whether we need to replace it with a real namespace or OLM would add here the one you specify during the installation.

Reply:
I think a good default like contour-operator for the namespace and then give the ability to override during installation